### PR TITLE
Designed the UI for productDetails

### PR DESCRIPTION
**Pull request by irenezhengg**
Current UI looks like this:

image

### CODE REVIEW

File: app/static/css/style.css [hunk 1]
```diff
@@ -13,3 +13,26 @@ a.mute-link:active { text-decoration: none; color: #212529; }
...
+
+.about {
+    font-size: 14px
+}
+
+.color {
+    margin-bottom: 10px
+}
+
+.view,body,html{height:100%}.carousel{height:50%}.carousel .carousel-inner,.carousel .carousel-inner .active,.carousel .carousel-inner .carousel-item{height:100%}@media (max-width:776px){.carousel{height:100%}}.page-footer{background-color:#929FBA}
```
Comment (Ming119) [inline app/static/css/style.css:38]: To make it easier to understand, can you split this line of css code?

File: app/templates/productDetails.html [hunk 2]
```diff
@@ -45,29 +45,25 @@
 {% else %}
 <div class="row my-5">
     <div class="col-md-6 mb-4 mb-md-0">
-        Column
+      <div class="col-md-10">
+          <div class="images p-3">
+              <div class="text-center p-4"> <img id="main-image" img src="https://api.lorem.space/image/shoes?w=304&amp;h=225" width="450" /> </div>
```
Comment (Ming119) [inline app/templates/productDetails.html:50]: The `img` tag can be changed as
```suggestion
              <div class="text-center p-4"> <img id="main-image" src="{{ product.img_url }}" /> </div>
```

**Review by Ming119 — CHANGES_REQUESTED** [review]

**Review by HuangShengYao108590061 — APPROVED** [review]
Modify based on Ming119 comments then it's ready to be merge to master

**Comment by irenezhengg** [comment]
@Ming119 the changes requested above have been done.

File: app/templates/productDetails.html [hunk 3]
```diff
@@ -45,29 +45,25 @@
 {% else %}
 <div class="row my-5">
     <div class="col-md-6 mb-4 mb-md-0">
-        Column
+      <div class="col-md-10">
+          <div class="images p-3">
+              <div class="text-center p-4"> <img id="main-image" src=" {{ product.img_url }} " width="450" /> </div>
```
Comment (Ming119) [inline app/templates/productDetails.html:50]: my mistake, should be `{{ product.image_url }}`

**Review by Ming119 — CHANGES_REQUESTED** [review]
@irenezhengg I think the remaining quantity of the product is also needed to show on page.

File: app/templates/productDetails.html (hunk 3 above)
Comment (irenezhengg) [reply app/templates/productDetails.html]: done

**Review by Ming119 — APPROVED** [review]

**Pull request merged**